### PR TITLE
chore(bigquery): prefer env var to arguments

### PR DESCRIPTION
**Pull request by vishwarajanand**
[Relevant cgc website](https://cloud.google.com/bigquery/docs/quickstarts/quickstart-client-libraries#complete_source_code)
[Relevant region tag](https://devrel.corp.google.com/snippets/bigquery_simple_app_all)

# Change
1. Remove the use of env var `GOOGLE_CLOUD_PROJECT` or command line args for running `bigquery/stackoverflow/stackoverflow.php`.
2. Relevant changes in test.

# Tests
1. Tests are passing in php74.
```
bigquery/stackoverflow % ../../testing/vendor/bin/phpunit -c phpunit.xml.dist            (git)-[chore_show_click_to_copy_code]-
PHPUnit 8.5.26 #StandWithUkraine

Error:         XDEBUG_MODE=coverage or xdebug.mode=coverage has to be set

.                                                                   1 / 1 (100%)

Time: 1.82 seconds, Memory: 8.00 MB

OK (1 test, 1 assertion)
bigquery/stackoverflow %
```

2. Tested that this samples executes without errors in cloudshell env:

Screenshot 2023-02-21 at 12 24 17

 _In continuation with: https://github.com/GoogleCloudPlatform/php-docs-samples/pull/1776_

### CODE REVIEW

**Comment by vishwarajanand** [comment]
CC: @chellebodnar-google

**Comment by vishwarajanand** [comment]
When I try to reproduce this in local, the lint checks pass. 😿

**Comment by vishwarajanand** [comment]
lint errors are fixed in: https://github.com/GoogleCloudPlatform/php-docs-samples/pull/1780

File: bigquery/stackoverflow/stackoverflow.php [hunk 1]
```diff
@@ -32,12 +32,11 @@
 # [END bigquery_simple_app_deps]
 
 // get the project ID as the first argument
-if (2 != count($argv)) {
-    die("Usage: php stackoverflow.php YOUR_PROJECT_ID\n");
+$projectId = getenv('GOOGLE_CLOUD_PROJECT');
+if (!$projectId) {
+    die("Usage: GOOGLE_CLOUD_PROJECT env var must be set.\n");
```
[the remark this answers is not in the document]
Comment (vishwarajanand) [reply bigquery/stackoverflow/stackoverflow.php:37]: @bshaffer Other language samples also expect this env var to be set. I agree with your argument from the repository maintenance POV. 

But if we continue with the script arguments, then users have no way to know about the extra param and we are expecting first time users to be able to just copy and run this script.

[the remark this answers is not in the document]
Comment (vishwarajanand) [reply bigquery/stackoverflow/stackoverflow.php]: Removed the project id and env var fetching altogether.

**Review by bshaffer — APPROVED** [review]

**Pull request merged**